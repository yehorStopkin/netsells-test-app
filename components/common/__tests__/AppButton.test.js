const { shallowMount } = require('@vue/test-utils');
const AppButton = require('@/components/common/AppButton').default;

describe('AppButton', () => {
    test('component with default props', () => {
        const wrapper = shallowMount(AppButton, { propsData: { text: 'test text' } });

        expect(wrapper.exists()).toBeTruthy();
        expect(wrapper.element).toMatchSnapshot();
    });
});
