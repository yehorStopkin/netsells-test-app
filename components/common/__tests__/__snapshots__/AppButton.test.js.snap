// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppButton component with default props 1`] = `
<div
  class="app-button"
  style="border-radius: 1; width: 100px; padding-top: 14px; padding-bottom: 19px; font-size: 20px;"
>
  
    test text

</div>
`;
